lib/params/order_type.js

```diff
@@ -0,0 +1,63 @@
+'use strict'
+
+const _isEmpty = require('lodash/isEmpty')
+const _includes = require('lodash/includes')
+const { Order } = require('bfx-api-node-models')
+const stringParam = require('./string')
+const matchWord = require('../util/match_word')
+
+const TYPES = Object
+  .keys(Order.type)
+  .map(k => !_includes(k, 'EXCHANGE') && !_includes(k, ' ') ? k : null)
+  .filter(k => !!k)
+  .map(k => (
+    _includes(k, '_')
+      ? k.split('_')
+      : [k]
+  ))
+  .sort((a, b) => b.length - a.length)
```

Comment:
https://github.com/bitfinexcom/bfx-api-node-models/blob/master/lib/order.js#L24-L28
https://github.com/bitfinexcom/bfx-api-node-models/blob/master/lib/order.js#L730
as there are keys with spaces and with underscores it's easy to use values instead
as for me this is more readable to understand
```js
_.map(
  _.reduce(Order.type, (acc, type) => {
    if (_.includes(type, 'EXCHANGE')) return acc
    return { ...acc, [type.replace('EXCHANGE ', '')]: type }
  }, {}),
  (type) => type.split(' ')
).sort((a, b) => b.length - a.length)
```